sdk/servicefabric/arm-servicefabric/src/operations/clusterVersions.ts

```diff
@@ -61,7 +61,7 @@ export class ClusterVersionsImpl implements ClusterVersions {
    */
   getByEnvironment(
     location: string,
-    environment: Enum14,
+    environment: ClusterVersionsEnvironment,
```

Comment:
feel like this may be not a breaking. because both the Enum14 and ClusterVersionsEnvironment are actually strings.